src/requireDatabaseInstance.ts

```diff
@@ -0,0 +1,34 @@
+import * as clc from "cli-color";
+import { getDefaultDatabaseInstance } from "./getDefaultDatabaseInstance";
+import * as utils from "./utils";
+
+/**
+ * Ensures that the supplied options have an instance set. If not, tries to fetch the default instance.
+ * @param options command options
+ */
+export async function requireDatabaseInstance(options: any) {
+  if (options.instance) {
+    return Promise.resolve();
+  }
+  try {
+    const instance = await getDefaultDatabaseInstance(options);
+    if (instance === "") {
+      return utils.reject(
```

Comment:
In an `async` function, prefer using `throw new FirebaseError`. The result should be the same, that the returned promise is rejected with the error.